src/pages/docs/continuous-integration/codemagic-ci.md

```
@@ -0,0 +1,92 @@
+> 
+>   - A **GitHub** repository
+>   - A **Codemagic** trial account
+>
+---
+
+## **Steps to Integrate Testsigma with Codemagic**
+
+1. Navigate to [Codemagic](https://codemagic.io/start/) and sign up using your **GitHub** account. 
+
+2. Click **Authorize Codemagic CI/CD**. 
+<img src="https://s3.amazonaws.com/static-docs.testsigma.com/new_images/projects/faq/Codemagic_2.png" style="width: 400px;" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)